app/jobs/creating_an_importer.md

```diff
@@ -0,0 +1,37 @@
+# How the importer works
```

Comment:
This is awesome. Will make it so much easier to get contribs from outside.

Reply:
yeah i think its quite confusing tbh, lots of hidden magic, i had to write all this out just to understand it. i would love to get your input on how to organise this code better. i dont think its far off but its the rare web problem that needs someone with a good understanding of class architecture to review, i think

Reply:
I think with this app, if we did decide to restructure fundamentally, it would be to port to a new framework or start from scratch with the data shapes... which would / should happen if we tried to scale or fundamentally provide a different service. I'm still kind of viewing it as a very Beta stage.

Reply:
i'm increasingly thinking that all this is all salvagable actually. i think theres a very strong case to migrate this bit of the app into a seperate gem (or other language), though. it would do a lot if we could just define what its meant to do more clearly and out of the context of placecal. it deffo doesnt make a lot of sense long term to have the importer and webapp running on the same server at the very least!